TraceScreen.c

```diff
@@ -90,11 +90,22 @@ bool TraceScreen_forkTracer(TraceScreen* this) {
 
       char buffer[32] = {0};
       xSnprintf(buffer, sizeof(buffer), "%d", this->super.process->pid);
-      execlp("strace", "strace", "-T", "-tt", "-s", "512", "-p", buffer, NULL);
 
-      // Should never reach here, unless execlp fails ...
-      const char* message = "Could not execute 'strace'. Please make sure it is available in your $PATH.";
-      (void)! write(STDERR_FILENO, message, strlen(message));
+      #if defined(HTOP_FREEBSD) || defined(HTOP_OPENBSD) || defined(HTOP_NETBSD) || defined(HTOP_DRAGONFLYBSD) || defined(HTOP_SOLARIS)
+         execlp("truss", "truss", "-s", "512", "-p", buffer, NULL);
+
+         const char* message = "Could not execute 'truss'. Please make sure it is available in your $PATH.";
+         (void)! write(STDERR_FILENO, message, strlen(message));
+      #elif defined(HTOP_LINUX)
+         execlp("strace", "strace", "-T", "-tt", "-s", "512", "-p", buffer, NULL);
+
+         // Should never reach here, unless execlp fails ...
+         const char* message = "Could not execute 'strace'. Please make sure it is available in your $PATH.";
+         (void)! write(STDERR_FILENO, message, strlen(message));
+      else // HTOP_PCP or HTOP_UNSUPPORTED
+         const char* message = "Tracing unavailable on not supported system.";
```

Comment:
```suggestion
         const char* message = "Tracing is unsupported for this system.";
```

Reply:
It's correct, since there's no such thing as strace or truss for MacOS. So it's unsupported

Reply:
In MacOS case it will print a message that such platform is unsupported, instead of showing a crash caused by "strace" unavailability (as it was before).